lib/cache/cache.go

```diff
@@ -223,8 +223,7 @@ func ForOldRemoteProxy(cfg Config) Config {
 		{Kind: types.KindAuthServer},
 		{Kind: types.KindReverseTunnel},
 		{Kind: types.KindTunnelConnection},
-		{Kind: types.KindAppServer},
-		{Kind: types.KindAppServer, Version: types.V2},
+		{Kind: types.KindAppServer, SubKind: types.SubKindTeleportApp},
```

Comment:
If we're connecting to a potentially old auth server, isn't this just effectively a watch for `KindAppServer`?